### ISSUE TITLE
[Question]: Are the images in this repo are the same ones used for the EKS control plane?

### ISSUE THREAD

**Issue by omerxx** [aws/eks-distro#3007]
Specifically - is the kube-scheduler image from [this public ECR repo](https://gallery.ecr.aws/eks-distro/kubernetes/kube-scheduler) is the same image used in the EKS control plane on AWS?

**Comment by rcrozean** [comment]
No, they are not the identical since managed eks has a couple additional patches for internal use only. Outside internal only changes they are the same.

**Comment by OverStruck** [comment]
yes
